cscs-checks/compile/libsci_resolve.py

```diff
@@ -67,14 +66,16 @@ def set_sanity(self):
 
 @rfm.simple_test
 class MKLResolveTest(LibSciResolveBaseTest):
-    def __init__(self):
-        super().__init__()
-        self.descr = '-mkl Resolves to MKL'
-        self.valid_prog_environs = ['PrgEnv-intel']
-        self.build_system = 'SingleSource'
+    descr = '-mkl Resolves to MKL'
+    valid_prog_environs = ['PrgEnv-intel']
```

Comment:
it fails with cdt/21.05: `cannot find -lmkl_intel_lp64`

It will build with:
```
/opt/intel/oneapi/modulefiles-setup.sh --output-dir=$PWD/m
module use $PWD/m
module load mkl
module list
```

Probably need to open an issue

Reply:
issue opened: https://portal.cray.com/apex/viewcase?Id=5005a00001oeRqaAAE
> CASE #285117 - MODULE INTEL/2021.2.0

Reply:
I have added the workaround with a FIXME